cap/modules/schemas/configs/notifications.json

```diff
@@ -0,0 +1,587 @@
+{
```

Comment:
Replace all schemas with a simple general JSON-Schema, as a placeholder for now. Till we close other relevant PRs that can replace these placeholders